### ISSUE TITLE
Add more detail to error returned from `loadConfig`

### ISSUE THREAD

**Issue by austinvalle** [bodgit/tsig#91]
Hello 👋🏻 ,

## Background 
We recently had some questions about a confusing error that bubbled up from this library in the DNS terraform provider: https://github.com/hashicorp/terraform-provider-dns/issues/128

```bash
Error updating DNS record: Error negotiating GSS context: configuration file could not be opened:  open : no such file or directory
```

This error is caused by not setting the `KRB5_CONFIG` environment variable and not having a valid config in one of the default locations defined here: https://github.com/bodgit/tsig/blob/10b91d46dbdc2db4fc524572ea95da64cb91358c/gss/gokrb5.go#L358

Looking a little deeper it seems in this situation that the blank file path is passed down to the `jcmturner/gokrb5` library and then fails in the `os.Open` method: https://github.com/jcmturner/gokrb5/blob/2a265cd1f9a5e200336857e55e38ef3c06815891/config/krb5conf.go#L522

## Proposal
- We could add more detail to the error message describing that no valid path was found in environment variable or default paths

**Comment by bodgit** [comment]
I've created #92 that updates that bit of code. I switch from `os.Getenv()` to `os.LookupEnv()` so I can distinguish between the environment variable existing or not. It should mean the following logic:
* If `KRB5_CONFIG` is set and the file it points to doesn't exist, error.
* Otherwise each list of default locations is tried, (currently a list of one). If the file doesn't exist, try the next location, any other error, return straight away including any errors from previous locations.
* If you reach the end with no file found, return a `multierror` containing the errors from all the locations tried.

This should mean `config.Load()` is never called with just `""`.

**Comment by austinvalle** [comment]
Thanks for looking into this @bodgit!

**Comment by bodgit** [comment]
If you're able to test out that branch feel free, otherwise I'll cut a new release once I'm satisfied it's not going to break anything.

**Comment by bodgit** [comment]
v1.2.2 has been released with the fix implemented in #92.

Thanks for the bug report!
